db/migrate/20211209165607_add_person_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddPersonToUser < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :users, :person, null: true, foreign_key: true
```

Comment:
actually necessary?

Reply:
What exactly do you mean? The `null: true` part?